### PR TITLE
Sublink colour

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Colours added to the kicker of the sublink

@GHaberis @SiAdcock @sndrs

### CODE REVIEW

**Review by GHaberis — APPROVED** [review]
👍

File: static/src/stylesheets/module/facia-garnett/_pillars.scss [hunk 1]
```diff
@@ -161,4 +161,9 @@ $pillars: (
     .fc-item--pillar-#{$pillar} {
         @include colours($palette);
     }
+    .fc-sublink.fc-sublink--pillar-#{$pillar} {
```
Comment (SiAdcock) [inline static/src/stylesheets/module/facia-garnett/_pillars.scss]: Is it possible to do this without specifying the `.fc-sublink` class in the selector? Or does it get overridden somewhere?

Comment (blongden73) [reply static/src/stylesheets/module/facia-garnett/_pillars.scss]: @SiAdcock This is done

**Review by SiAdcock — APPROVED** [review]
Love it, thanks @blongden73 ! ❤️

**Pull request merged**